setup.sh

```diff
@@ -127,6 +128,7 @@ function install_mac_newer_python_dependencies() {
         echo "-------------------------"
         brew install c-blosc
     fi    
+    export CBLOSC_DIR=$(brew --prefix)
```

Comment:
I kinda have to be blunt and ask @samgermain to have a look as i'm no MacOS user - which means i kinda struggle determining if this is right or wrong (i trust it's right, but another opinion won't hurt).

Reply:
@xmatthias I think it's probably fine, I uninstalled `hdf5` and `c-blosc` and removed `.env` and the script worked fine, I'm not really sure how to test it other than that though

Reply:
That's more than i am able to do without a mac :+1: